### PR TITLE
[BER-FTWD-082019] - Thi & Jan

### PR DESCRIPTION
**Pull request by janRedmann**

### CODE REVIEW

File: starter-code/src/functions-and-arrays.js [hunk 1]
```diff
@@ -1,26 +1,49 @@
...
+  let sum = 0;
+  for (let i=0; i < numbers.length; i++) {
+    sum += numbers[i];
+  }
+  return sum;
+ }
+ // Calculate the Average
+ var numbersAvg = [2, 6, 9, 10, 7, 4, 1, 9];
+ function numbersAvg (numbers) {
+  let sum = 0;
```
Comment (mjarraya) [inline starter-code/src/functions-and-arrays.js]: Reuse the `sumArray` function!

File: starter-code/src/functions-and-arrays.js [hunk 2]
```diff
@@ -1,26 +1,49 @@
...
+ function sumArray (numbers) {
+  let sum = 0;
+  for (let i=0; i < numbers.length; i++) {
+    sum += numbers[i];
+  }
+  return sum;
+ }
+ // Calculate the Average
+ var numbersAvg = [2, 6, 9, 10, 7, 4, 1, 9];
+ function numbersAvg (numbers) {
```
Comment (mjarraya) [inline starter-code/src/functions-and-arrays.js]: `averageNumbers` ?

File: starter-code/src/functions-and-arrays.js [hunk 3]
```diff
@@ -1,26 +1,49 @@
...
-// Calculate the Average
-
-var numbersAvg = [2, 6, 9, 10, 7, 4, 1, 9];
-
-// Array of Strings
-var wordsArr = [
+ ];
+ function findLongestWord (words) {
+ let longestWord = words[0];
+ for (let i=1; i < words.length; i++) {
```
Comment (mjarraya) [inline starter-code/src/functions-and-arrays.js]: smart!

File: starter-code/src/functions-and-arrays.js (hunk 2 above)
Comment (janRedmann) [reply starter-code/src/functions-and-arrays.js]: 🤦‍♂️ - corrected that in the console but didn't update the file

**Pull request closed without merging**